app/src/main/res/xml/widget_preferences.xml

```diff
@@ -22,94 +22,112 @@
   ~ SOFTWARE.
   -->
 
-<PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
+<!--suppress AndroidElementNotAllowed -->
+<androidx.preference.PreferenceScreen xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
-    <PreferenceCategory android:title="@string/pref_schedule_title">
-        <SwitchPreference
+    <androidx.preference.PreferenceCategory android:title="@string/pref_schedule_title">
+        <androidx.preference.SwitchPreferenceCompat
             android:key="show_all_times"
             android:summaryOff="@string/pref_show_all_times_off"
             android:summaryOn="@string/pref_show_all_times_on"
-            android:title="@string/pref_show_all_times" />
+            android:title="@string/pref_show_all_times"
+            app:iconSpaceReserved="false" />
```

Comment:
Do we need this after the resources fix?
https://github.com/AngelMariages/RodaliesWidget/blob/457ed0e001b79c6530886c0312e43b47f025cc7d/app/src/main/res/values-sw360dp-v13/values-preference.xml